mlflow/tracking/fluent.py

```diff
@@ -140,6 +140,11 @@ def start_run(run_id=None, experiment_id=None, run_name=None, nested=False):
                 "Cannot start run with ID {} because it is in the "
                 "deleted state.".format(existing_run_id)
             )
+        end_time = active_run_obj.info.end_time
+        MlflowClient().get_store().update_run_info(
+            existing_run_id, run_status=RunStatus.RUNNING, end_time=end_time
```

Comment:
In the docs, for `start_run`, can we indicate that the run status is set to `RunStatus.RUNNING` when resuming a run?

Also, we discussed this change with other team members, right? Just making sure everyone's on board here (it makes sense to me!).

Reply:
Yep, I had brought it up a while ago as well :) Will write docs change.